src/eva/data/mon_data_space.py

```diff
@@ -0,0 +1,283 @@
+# (C) Copyright 2021-2023 NOAA/NWS/EMC
+#
+# (C) Copyright 2021-2023 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+import os
+import numpy as np
+
+from xarray import Dataset, concat
+from scipy.io import FortranFile
+from datetime import datetime
+
+from eva.eva_base import EvaBase
+from eva.utilities.config import get
+from eva.utilities.utils import parse_channel_list
+
+# --------------------------------------------------------------------------------------------------
+
+
+class MonDataSpace(EvaBase):
+
+    # ----------------------------------------------------------------------------------------------
+
+    def execute(self, data_collections, timing):
+
+        # Get name of the control file to parse
+        # -------------------------------------
+        ctlfile = self.config.get('ctlfile')
```

Comment:
perhaps change to `control file` to be more explicit/clear?

Reply:
Done.